backend/utils.py

```diff
@@ -42,3 +45,31 @@ async def write_md_to_pdf(text: str) -> str:
 
     encoded_file_path = urllib.parse.quote(f"{file_path}.pdf")
     return encoded_file_path
+
+async def write_md_to_word(text: str) -> str:
```

Comment:
How are you planning to use it? Should we maybe add a config flag that if True will also download a docx version? Or have it in the UI for selection? @proy9714

Reply:
I was hoping someone could contribute and make a UI button that downloads the word document if clicked.